charts/sck-otel/values.yaml

```diff
@@ -132,6 +132,22 @@ containerLogs:
   # Multiline logs currently support only container name which is not unique within the cluster.
   # Can we also support k8s object owner name (deployment, daemonset, statefulset) along with namespace?
   multilineConfigs: []
+  # Example configuration for processing and handling multiline logs/ java stack trace from test container "buttercup-app-python" by specifying  a regular expression (first_entry_regex) to parse first entry in a multiline log series.
+  # Sample python stack trace/ multiline log from container "buttercup-app-python"
+  # .........
+  # Traceback (most recent call last):
+  #   File "/path/to/example.py", line 3, in <module>
+  #     greet(1)
+  #   File "/path/to/greetings.py", line 5, in greet
+  #     print(greeting + ', ' + who_to_greet(someone))
+  # TypeError: must be str, not int
+  # .........
+  #
+  # Sample configuration to handle multiline python stack trace from buttercup-app-python container
+  # multilineConfigs:
+  #  - containerName: buttercup-app-python
+  #    first_entry_regex: ^Traceback \(most recent call last\):$
```

Comment:
could you elaborate a little more?
This just gives a regex for a stacktrace pattern example like above :)

Reply:
so python would print the stacktrace only when there is error and it is terminating its process. but for all INFO logs that app would normally print out, it doesn't match the first entry regex. so it would be constantly merged into one event until flush time hits.